charts/cp-schema-registry/templates/schema-registry-ui.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  {{- if .Values.ui.annotations }}
+  annotations: {{ toYaml .Values.ui.annotations | nindent 4 }}
+  {{- end}}
+  labels:
+    app: {{ template "cp-schema-registry.fullname" . }}
+  {{- if .Values.ui.labels }}
+  {{ toYaml .Values.ui.labels | indent 4}}
+  {{- end}}
+  name: {{ template "cp-schema-registry.fullname" . }}-ui
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "cp-schema-registry.fullname" . }}
+      service: schema-registry-ui
+  template:
+    metadata:
+      annotations:
+      {{- if .Values.ui.annotations }}
+      {{ toYaml .Values.ui.annotations | indent 6 }}
+      {{ end }}
+      labels:
+        app: {{ template "cp-schema-registry.fullname" . }}
+        service: schema-registry-ui
+      {{- if .Values.ui.labels }}
+      {{ toYaml .Values.ui.labels | indent 8}}
+      {{- end}}
+    spec:
+      securityContext:
+        runAsUser: 10001
+        runAsGroup: 10001
+        fsGroup: 10001
+        runAsNonRoot: true
+      containers:
+        - env:
+            - name: PROXY
+              value: "true"
+            - name: SCHEMAREGISTRY_URL
+              value: {{ template  "cp-schema-registry.connectUrlValue" . }}
+          image: landoop/schema-registry-ui
+          imagePullPolicy: IfNotPresent
+          name: {{ template "cp-schema-registry.fullname" . }}-ui
```

Comment:
The name of the container is variable ?
Maybe you should just use a simple name like "ui" (the name only needs to be unique among the containers of the pod AFAIK, so here 1 container, it's simple)

Reply:
Yes it could be, juste to be sure, do you see an improvement by removing the variable part or is it just a suggestion in order to have a shorter name ?

Reply:
My idea was: avoid using a "complex" Helm syntax for something that can be hardcoded
=> for better readability and maintainability of the Chart

Reply:
I see, thanks for the explanation and for the review! will update this